src/utils/openai.ts

```diff
@@ -4,19 +4,31 @@ import fetchContext from '@/utils/fetchContext';
 import { getChatResponse } from './handleApiRequest/getChatResponse/getChatResponse';
 import createEmbedding from './handleApiRequest/createEmbedding/createEmbedding';
 
-export const callOpenai = async (userQuestion: string) => {
+export type CallOpenaiType = {
+  chatMessage: string;
+  associatedUrl: string;
+};
+
+export const callOpenai = async (
+  userQuestion: string
+): Promise<CallOpenaiType> => {
   const embedding = await createEmbedding(userQuestion);
   const contextResponse = await fetchContext(embedding);
 
   if (contextResponse.length === 0) {
     console.warn(
       `Warning: Could not find any similar embeddings from the database relating to the question "${userQuestion}".`
     );
-    return 'Unable to find information on this topic.';
+    return {
+      chatMessage: 'Unable to find information on this topic.',
+      associatedUrl: '',
```

Comment:
What's displayed in this case? An empty link?

Reply:
The 'You can find more information at this link' section isn't displayed :)

Reply:
Perfect